docs/data/joy/components/modal/modal.md

```diff
@@ -127,35 +140,37 @@ declare module '@mui/joy/ModalDialog' {
 }
 ```
 
-#### Variant
-
-The `ModalDialog` supports the [global variants](/joy-ui/main-features/global-variants/) feature.
+#### Vertical scroll
 
-The `ModalClose`'s variant adapts automatically to have a proper contrast to the `ModalDialog`.
+By default, content within the `ModalDialog` won't overflow the screen when its height is bigger than the viewport.
 
-{{"demo": "VariantModalDialog.js"}}
+To ensure your content is visible, make the container holding it overflow by adding the [`overflow` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) with either `scroll` or `auto` values.
 
-#### Size
+{{"demo": "DialogVerticalScroll.js"}}
 
-The `ModalDialog` comes with 3 sizes, `sm`, `md` (default), and `lg`.
+### Modal overflow
 
-The `ModalClose` and `ModalDialogTitle` inherits the size from the `ModalDialog` unless it is specified in each component directly.
+The previous section demonstrated how to make content _within_ the modal scrollable.
 
-{{"demo": "SizeModalDialog.js"}}
+To make the _whole_ modal scrollable, in case its higher than the viewport, use the `ModalOverflow` component.
+It will allow the `ModalDialog` to vertically overflow the screen.
 
-### Alert Dialog
+It supports both `center` and `fullScreen` built-in layouts.
```

Comment:
What does "It" refer to here? Modal Dialog, or Modal Overflow?

Reply:
It's referring to the Modal Overflow!

Reply:
Should it be explicit using "Modal Overflow"?